pnpm-workspace.yaml

```diff
@@ -36,7 +37,6 @@ catalogs:
     unbuild: ^3.6.0
     webpack: ^5.101.0
     webpack-sources: ^3.3.3
-
```

Comment:
So is this empty line deleted automatically after installation? If not, I want to keep it.